vpn/tunnel.go

```diff
@@ -97,6 +116,20 @@ func (t *Tunnel) requestLoop() {
 	}
 }
 
+func (t *Tunnel) netStatusLoop() {
+	ticker := time.NewTicker(netStatusInterval)
```

Comment:
this should use `quartz.Clock.TickerFunc()`, which should make unit testing easier.